components/grid/paging.md

```diff
@@ -12,11 +12,11 @@ position: 20
 
 The Grid component offers support for paging.
 
-To enable paging, set its `Pageable` property to `true`. 
+To enable paging, set the Grid `Pageable` property to `true`. 
 
-You can control the number of records per page through the `PageSize` property.
+You can control the number of records per page through the Grid `PageSize` property.
 
-You can set the current page of the grid through its integer `Page` property.
+You can set the current page of the Grid through its integer `Page` property.
 
```

Comment:
Consider mentioning the advanced settings section here in the intro, so people quickly see there is more than page size.